1.16/Dockerfile.fedora

```diff
@@ -0,0 +1,83 @@
+FROM registry.fedoraproject.org/f29/s2i-core:latest
+
+# nginx 1.14 image.
+#
+# Volumes:
+#  * /var/log/nginx/ - Storage for logs
+
+EXPOSE 8080
+EXPOSE 8443
+
+ENV NAME=nginx \
+    NGINX_VERSION=1.14 \
```

Comment:
nginx1.16 actually is in f29 already, so the dockerfile can be changed to 1.16, but it would be also good to use non-EOL Fedora version, not sure the updated s2i images for f30/f31are already in stable though.